build/configs/rtl8730e/cpp_loadable_8M/defconfig

```diff
@@ -112,7 +112,7 @@ CONFIG_ARCH_LOWVECTORS=y
 #
 # CONFIG_ARCH_HAVE_DABORTSTACK is not set
 CONFIG_STACK_OVERFLOW_PROTECTION_DISABLE=y
-# CONFIG_SYSTEM_REBOOT_REASON is not set
+CONFIG_SYSTEM_REBOOT_REASON=y
```

Comment:
This config is set to No by default in https://github.com/Samsung/TizenRT/blob/23df385e3eea438568f8c0158a48c4d4eca319f5/os/arch/arm/Kconfig#L468
Please consider updating the Kconfig, maybe on the basis of the selected chipset, so that a menuconfig save does not overwrite this change,

Reply:
Thank you for opinion, 
Is your opinion creating ARCH_HAVE_REBOOT_REASON configuration?